src/verilog/expr2verilog.cpp

```diff
@@ -2074,6 +2074,12 @@ std::string expr2verilogt::convert(const typet &type)
   {
     return "enum";
   }
+  else if(type.id() == ID_verilog_queue)
+  {
+    std::string dest = "queue of ";
+    dest += convert(to_type_with_subtype(type).subtype());
```

Comment:
Is it intentional that this will not produce valid SystemVerilog?

Reply:
Yes; since you need a declarator, there's not parse-able SystemVerilog just for the type.